CHANGELOG.md

```diff
@@ -5,6 +5,20 @@ All notable changes to `model_bakery` will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased](https://github.com/model-bakers/model_bakery/tree/master)
+
+### Added
+
+### Changed
+- Improve code comments (https://github.com/model-bakers/model_bakery/pull/31)
+- Switch to tox-travis (https://github.com/model-bakers/model_bakery/pull/43)
+- Add black job (https://github.com/model-bakers/model_bakery/pull/42)
+- README.md instead of rst (https://github.com/model-bakers/model_bakery/pull/44)
```

Comment:
Thanks for adding this! By the way, we could setup a GitHub Action for CHANGELOGs. :innocent:

Reply:
+1

Reply:
https://github.com/model-bakers/model_bakery/issues/50